src/components/CheckListItem/CheckTypePill.tsx

```diff
@@ -10,11 +9,9 @@ interface Props {
 }
 
 export const CheckTypePill = ({ checkType, onClick, className }: Props) => {
-  const theme = useTheme();
-
   return (
     <Pill
-      color={theme.palette.blue77}
+      color="rgb(110, 159, 255)"
```

Comment:
This is the primary button blue, I couldn't find it in our theme color constants but I'm sure it exists somewhere.

Reply:
The only places where I find that are:

```
packages/grafana-data/src/themes/palette.ts:  blueDarkText: '#6E9FFF', // '#58a6ff', //'#33a2e5', // '#5790FF',
public/sass/_variables.dark.generated.scss:$blue-light: #6E9FFF;
public/sass/_variables.dark.generated.scss:$variable: #6E9FFF;
public/sass/_variables.dark.generated.scss:$text-blue: #6E9FFF;
public/sass/_variables.dark.generated.scss:$external-link-color: #6E9FFF;
public/sass/_variables.dark.generated.scss:$input-border-focus: #6E9FFF;
public/sass/_variables.dark.generated.scss:$input-box-shadow-focus: #6E9FFF !default;
```